src/app/post/post.html

```diff
@@ -201,9 +201,7 @@ <h4><i class="fa fa-shield"></i> <strong>Uncensorable</strong></h4>
                   ng-class="{'bold': postCtrl.responseSortOrder === 'createdAt'}">chronologically</a>
               </span>
             </div>
-            <a ng-if="postCtrl.post.status !== 'archived'" target="_self" ng-href="{{
-                user.id ? '/editor/comment/' + postCtrl.post.id : '/signup'
-              }}">
+            <a ng-if="postCtrl.post.status !== 'archived'" target="_self" ng-click="postCtrl.redirectUserToComments()">
```

Comment:
This method is private. It only works because the typescript compiles all private method to public anyway. A more elaborated compiler would throw an error here.

General principle: if the method is used in a template, it should be public.